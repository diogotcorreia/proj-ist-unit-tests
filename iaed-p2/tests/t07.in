set proj2.c resolução do projeto
set README.md enunciado do projeto
set tests/Makefile execução dos testes
set tests/test01.in primeiro teste
set tests/test02.in segundo teste
set Makefile construção do projeto
set guidelines.md orientações para a escrita do projeto
search segundo teste
quit
