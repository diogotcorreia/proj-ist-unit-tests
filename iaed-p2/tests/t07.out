/tests/test02.in
